tests/test_timeit_logger.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import print_function
+import logging
+import unittest
+from mock import patch
+import profilehooks
+import sys
+import time
+
+
+class testProfilehooksTimeitLogger(unittest.TestCase):
+
+    def sample_fn(self, delay_time, delay=False):
+        if delay:
+            time.sleep(delay_time)
+        return delay
+
+    def test_timecall_with_logger(self):
+        if sys.version_info < (2, 7):
+            print('Test does not support python 2.6')
```

Comment:
Why not?  Because of `assertIn`?  Or did `mock` drop Python 2.6 support already?

Lets remove this check and just drop 2.6 support for profilehooks entire.